protocol/contracts/oracle/Cargo.toml

```diff
@@ -80,5 +61,6 @@ currency = { workspace = true, features = ["testing"] }
 finance = { workspace = true }
 platform = { workspace = true, features = ["testing"] }
 schema = { workspace = true }
+oracle = { workspace = true, features = ["contract"] }
```

Comment:
Are you sure the oracle depends on itself?

Reply:
These are the `dev-dependencies` so I'm pretty confident there was a reason to add it too. I think the tests required some `contract`-gated modules/definitions.